topeft/modules/tmp_corrections.py

```diff
@@ -0,0 +1,1198 @@
+
```

Comment:
What is this file for?

Reply:
It was a temporary script made by Amanda, I removed it from the GH repo